src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultRazorConfigurationService.cs

```diff
@@ -52,7 +53,8 @@ public async override Task<RazorLSPOptions> GetLatestOptionsAsync()
                     }
                 };
 
-                var result = await _server.Client.SendRequest<ConfigurationParams, object[]>("workspace/configuration", request);
+                var response = _server.SendRequest<ConfigurationParams>("workspace/configuration", request);
+                var result = await response.Returning<object[]>(CancellationToken.None);
```

Comment:
this cancellation token should probably come from the caller. I understand that `GetLatestOptionsAsync` doesn't currently take a cancellation token (it should)